index.html

```diff
@@ -2,35 +2,48 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <meta name="theme-color" content="#6c4bc1" />
     <title>micro:bit Python Editor</title>
     <meta property="og:title" content="micro:bit Python Editor" />
+    <link rel="canonical" href="%VITE_FULL_URL%" />
+    <meta property="og:url" content="%VITE_FULL_URL%" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@microbit_edu" />
     <meta
       property="og:description"
       content="Built by the Micro:bit Educational Foundation and the global Python Community."
     />
-    <meta
-      property="og:image"
-      content="https://python.microbit.org%PUBLIC_URL%/social.png"
-    />
-    <meta name="twitter:card" content="summary_large_image" />
     <meta
       name="twitter:description"
       content="A Python Editor for the BBC micro:bit, built by the Micro:bit Educational Foundation and the global Python Community."
     />
-    <% if (process.env.REACT_APP_FOUNDATION_BUILD === 'true') { %>
-    <script src="https://shared-assets.microbit.org/common/v1/common.js" async></script>
-    <link rel="stylesheet" href="https://shared-assets.microbit.org/common/v1/main.css"></link>
+    <meta property="og:image" content="%VITE_FULL_URL%social.png" />
```

Comment:
I've checked this on the review branch: https://review-python-editor-v3.microbit.org/vite/social.png